crates/tests-e2e/src/logger.rs

```diff
@@ -8,13 +8,18 @@ pub fn init_tracing() {
             .with_env_filter(
                 "debug,\
                  hyper=warn,\
-                 reqwest=warn,\
+                 reqwest=info,\
                  rustls=warn,\
                  bdk=info,\
                  lightning::ln::peer_handler=debug,\
                  lightning=trace,\
+                 lightning_background_processor=info,\
+                 lightning_transaction_sync=warn, \
                  sled=info,\
-                 ureq=info",
+                 sled::pagecache::iobuf=info,\
```

Comment:
🤔I'm pretty sure you need to move the most specific names to the top for them to take effect. E.g.

```
lightning=trace,\
lightning_background_processor=info,\
```

keeps `lightning_background_processor`  on TRACE.